main_test.go

```diff
@@ -207,12 +214,33 @@ func checkStatusData(t *testing.T, fixture Fixture, endpoint string, results Res
 	// check the configuration
 	wantConf := fixture.Expect.Config.ToStringMap()
 	gotConf := results.Config.ToStringMap()
+	// if an expected config string is set to "*" it will match anything
+	// and is effectively ignored
+	for k, v := range wantConf {
+		if v == "*" {
+			// set to same so cmd.Diff will ignore
+			wantConf[k] = gotConf[k]
+		}
+	}
```

Comment:
As we discussed elsewhere, I think if `*` becomes a problem the next step is to switch those strings to require a *regexp.Regexp. This would be powerful, and a bit sharper. Perhaps the first thing to explore is if some of those "*" use cases could be clearer and more precise, e.g. `Expect.TraceId = "*"` becomes 

```go
Expect.TraceId = regexp.MustCompile(`^[0-9a-fA-F]{32}$\`)
```

I'm not sure it's worth it yet but it'd be cool to explore as the test suite grows. An issue would be cool, it's a good question and I want to get the test suite to a spot where we feel confident releasing automatically.